### PR TITLE
Fix Accuracy test failure for gemma-4-31B-it

### PR DESCRIPTION
**Pull request by yiw-wang**
The upstream vllm `vllm/model_executor/models/gemma4_mm.py` expecting a `torch.Tensor` rather than a python bool.

This will fix the failures shown up in the nightly runs. e.g. https://buildkite.com/tpu-commons/tpu-inference-ci/builds/18558/canvas?jid=019e688e-484e-4595-959e-7729dd7e52da&tab=output

### Test

https://buildkite.com/tpu-commons/tpu-inference-ci/builds/18907/canvas

### CODE REVIEW

**Review by QiliangCui — APPROVED** [review]

File: tpu_inference/models/vllm/vllm_model_wrapper.py [hunk 1]
```diff
@@ -515,8 +515,8 @@ def embed_input_ids_func(
                         "call_args": call_args,
                         "call_kwargs": {
                             "is_multimodal":
-                            torch_view(is_multimodal)
-                            if is_multimodal is not None else False,
+                            torch_view(is_multimodal) if is_multimodal
+                            is not None else torch.tensor(False),
```
Comment (lk-chen) [inline tpu_inference/models/vllm/vllm_model_wrapper.py]: actually it should be `... else None`
or even better
```
"call_kwargs": {
  "is_multimodal": torch_view(..)
} if is_multimodal is not None else {}
```

Comment (yiw-wang) [reply tpu_inference/models/vllm/vllm_model_wrapper.py]: suggestion applied, thanks!

**Review by lk-chen — APPROVED** [review]

**Review by gxd3 — APPROVED** [review]
